doc-surrealdb_versioned_docs/version-latest/surrealql/statements/define/event.mdx

```diff
@@ -106,3 +118,37 @@ The `OVERWRITE` clause can be used to define an event and overwrite an existing
 -- Create an EVENT and overwrite if it already exists
 DEFINE EVENT OVERWRITE example ON example THEN {};
 ```
+
+## Events and permissions
+
+Queries inside the event always execute without any permission checks, even when triggered by changes made by the currently authenticated user. This can be very useful to perform additional checks and changes that involve tables/records that are inaccessible for the user.
+
+Consider a CREATE query send by a record user that has CREATE access to the `comment` table only:
```

Comment:
Thanks for the PR! All I see requiring a change is this "send"  which should be "sent". (Or just remove the word)

Reply:
Good catch!